app/routes/deploy.py

```diff
@@ -55,10 +55,26 @@ async def deploy():
         entry["image"] = "bitswan/pipeline-runtime-environment:latest"
 
         network_mode = None
+        secret_groups = []
         if pipeline_conf:
             network_mode = pipeline_conf.get(
                 "docker.compose", "network_mode", fallback=conf.get("network_mode")
             )
+            secret_groups = pipeline_conf.get("secrets", "groups", fallback="").split(
+                " "
+            )
+        for secret_group in secret_groups:
+            gitops_dir = os.environ.get(
+                "BITSWAN_GITOPS_DIR",
+                os.path.join(os.environ.get("HOME"), ".config/bitswan/local-gitops/"),
+            )
+            if os.path.exists(gitops_dir):
+                secret_env_file = os.path.join(gitops_dir, "secrets", secret_group)
+                if os.path.exists(secret_env_file):
```

Comment:
Same like above...I think we should mount the gitops secrets folder from host to gitops container and we should specify the path for both host and inside the container...the gitops will then check if file with secrets exists inside the container but the `env_file` in pipeline container will be host path

Reply:
Okay my bad..I tested it locally and it seems that the env_file is being loaded from the gitops container before the pipeline container actually starts...so this usage of container path is correct